bazaarvoice/emodb
# Certshovel

## Github Issue #

PD-233234

## What Are We Doing Here?

Increasing the timeout and stabilising emo build 

## How to Test and Verify

Already deployed to CI, CERT in. both the region

1. Check out this PR
2. Run Command X, Click Button Y
3. Profit

## Risk

Emo build need to stabilise and kodak timeout related changes in blob client

### Level 

`Medium`

### Required Testing

`Smoke`, `Regression`, or `Manual`. (All changes except documentation need smoke
testing at a minimum).

### Risk Summary

Add one or a few complete sentences about the possible risks or concerns for
this change.

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
